docs/WhatAndWhy/security.md

```diff
@@ -11,15 +13,71 @@ were required when authoring the patch are described in the patch notes).
 tag in the commit title that indicates the related bug database tracking the issue (e.g. TianoCore Bugzilla, CVE, etc.).
 
-## Requesting Access
+### Requesting Access
 
 Due to a number of factors including disclosure embargoes, these repos are available to the public by request only. If
 you are interested in getting access, please contact us through the Project Mu Team. Access will be granted to any
 members who can establish that they have already been given prior clearance to the relevant bug tracking databases.
 
-## Usage
+### Usage
 
 The Security Patch repos maintain branches that are 1:1 matches with public branches. All that is necessary to ingest
 the security patches is to re-target your Git submodule at the Security Patch repo and pull the branch with the same name.
 They are also kept in sync with any `*_RC` tags that are applied to the public repo, so it should be simple to identify
 which public commit corresponds to which SP repo commit.
+
+## Active Security Patch Auditing
+
+In project Mu's public repos a `SecurityFixes.yaml` file will exist in any package that CVE's have been entered against.
+This yaml file will be both human and machine readable to enable both human inspection and scripting. These files will
+allow a firmware developer to ensure that the firmware they are building is up to date with the latest CVE's. In addition
+to what is in the public, the embargoed fixes described above will also have their own entries in the Security repos.
+
+```yaml
+UNIQUE_NAME:
+# In most scenarios this will be the CVE, however may be whatever is deemed clearest to the consumer
+    
+    commit_titles: 
+    # This section contains the commit titles that patched the CVE, this allows for finding the commits even if 
+    # the repo was rebased. If your firmware is patched your git log will contain these git messages
+
+        - # Upstreamed Tianocore EDK2 Patch
+        # Most CVE's will only have a single, patch but additional entries may be used to confirm that a CVE has
+        # been patched by any of the patches. An example of this is when a CVE has been patched internally 
+        # and the upstream repo has chosen a different patch.
+          
+            - 'SecurityPkg: Example commit message fixing code related to the CVE' 
+            - 'SecurityPkg: An additional commit message fixing a different section of code'
+        - # Internal patch
+            - 'SecurityPkg: Our version of the patch that chose to take'
+    
+    cve:
+    # This is the CVE being patched
+        CVE-XXXX-YYYYY
+    
+    date_reported:
+    # This is the date the CVE was first reported
+        1970-01-01
+    
+    description:
+    # A short description of the issue take from the security issue
+        'imporper bounds checking on user data for <feature>'
+    
+    notes:
+    # An optional section where a contributor make notes they want consumers to see
+        - This is a note
+
+    files_impacted:
```

Comment:
I'm thinking this is less and less useful, if we know the commit it was supposed to be in is it necessary to list every file?